packages/react-router/lib/react-router-routing-provider.ts

```diff
@@ -0,0 +1,31 @@
+import { type OnSettle, createDefaultRoutingProvider } from '@bugsnag/browser-performance'
+import { matchPath } from 'react-router-dom'
+
+export interface RouteObject {
+  path?: string
+  children?: RouteObject[]
+}
+
+function flattenRoutes (routes: RouteObject[], _prefix: string = ''): string[] {
+  const prefix = `${!_prefix || _prefix === '/' ? _prefix : `${_prefix}/`}`
+  return [
+    ...routes.map(route => `${prefix}${route.path || ''}`),
+    ...routes.reduce<string[]>(
+      (accum, route) => [...accum, ...(route.children ? flattenRoutes(route.children, `${prefix}${route.path}`) : [])],
+      []
+    )
+  ]
+}
+
+export const createReactRouterRoutingProvider = (onSettle: OnSettle, location: Location) => {
+  const DefaultRoutingProvider = createDefaultRoutingProvider(onSettle, location)
```

Comment:
do you need to re-create a default routing provider here, or can you just import `DefaultRoutingProvider` from browser and extend it?